lua/wikis/commons/Infobox/Person.lua

```diff
@@ -446,6 +540,49 @@ function Person:displayLocations()
 	end)
 end
 
+---@param roleKey string
+---@return PersonRoleData?
+function Person:_createRoleData(roleKey)
+	if String.isEmpty(roleKey) then return nil end
+
+	local key = roleKey:lower()
+	local roleData = ROLES[key]
+
+	if roleData then return roleData end
+
+	local display = String.upperCaseFirst(roleKey)
+	return {
+		display = display,
+		category = display .. 's'
+	}
+end
+
+---@param roleData PersonRoleData?
+---@return string?
+function Person:_displayRole(roleData)
+	if not roleData then return end
+
+	---@param postFix string|integer|nil
+	---@return string?
+	local toDisplay = function(postFix)
+		postFix = postFix or ''
+		if not roleData['category' .. postFix] then return end
+		return Page.makeInternalLink(roleData['display' .. postFix], ':Category:' .. roleData['category' .. postFix])
+	end
+
+	local displays = Array.append({},
+		toDisplay(),
+		toDisplay(2),
+		toDisplay(3)
+	)
```

Comment:
PersonRoleData doesn't have fields `category2` and `category3`.
If we want to support this in the future, it would be way cleaner to implement as array instead, as these aren't read from wikicode but defined in lua.